notebooks/search/07-inference.ipynb

```diff
@@ -0,0 +1,382 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7a765629",
+   "metadata": {},
+   "source": [
+    "# Semantic Search using the Inference API\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/search/07-inference.ipynb)\n",
+    "\n",
+    "\n",
+    "Learn how to use the [Inference API](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) for semantic search."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c99b06d",
+   "metadata": {},
+   "source": [
+    "# 🧰 Requirements\n",
+    "\n",
+    "For this example, you will need:\n",
+    "\n",
+    "- An Elastic deployment with minimum **4GB machine learning node**\n",
```

Comment:
I'm guessing you copied this 4GB requirement from some older notebook? We have removed these requirements from other notebooks in this directory, mainly because we determined that everything worked fine using default choices for size, so it is an unnecessary complication to force users to check what size they've got.